logic/a_logic.go

```diff
@@ -330,6 +330,15 @@ func ConvertUserData(flag string, remoteData []map[string]interface{}) (users []
 func InitCron() {
 	c := cron.New(cron.WithSeconds())
 
+	if config.Conf.System.EnableCheckUser {
+		_, err := c.AddFunc(config.Conf.System.CheckUserSyncTime, func() {
+			User.CheckUserTime(nil, nil)
+		})
+		if err != nil {
+			common.Log.Errorf("eteststset: %v", err)
+		}
```

Comment:
在定时任务错误处理中，日志输出的错误信息"eteststset"看起来像是测试代码残留，应该改为更有意义的错误描述。

```suggestion
common.Log.Errorf("Failed to add user expiration check cron job: %v", err)